operator/pkg/gateway-api/gateway_reconcile.go

```diff
@@ -143,14 +144,16 @@ func (r *gatewayReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		return fail(err)
 	}
 
+	setGatewayReady(gw, true, "Gateway successfully configured")
```

Comment:
Hmm, Ready is now deprecated, actually, so we shouldn't set it.

Reply:
Removing it :)